src/accommodation/accommodation.service.ts

```diff
@@ -293,34 +312,24 @@ export class AccommodationService {
 
       skip: (options.page! - 1) * options.limit!,
       take: options.limit,
-
-      orderBy: [],
     };
 
-    if (options.orderByPeople) {
-      findManyOptions.orderBy.push({
-        [OrderBy.NUMBER_OF_PEOPLE]: options.orderByPeople,
-      });
-    }
-
-    if (options.orderByPrice) {
-      findManyOptions.orderBy.push({
-        [OrderBy.PRICE]: options.orderByPrice,
-      });
-    }
+    const orderingBy = this.generateOderingArray<Record<OrderBy, string>>([
+      [options.orderByPeople, OrderBy.NUMBER_OF_PEOPLE],
```

Comment:
spread 'options' on very top of this function to avoid 'options.' reference so many times